web/themes/custom/pineapple/sass/components/paragraphs/_content-spotlight.scss

```diff
@@ -0,0 +1,5 @@
+.spotlight-heading {
```

Comment:
Why do you need this file? I thought all your headers, <h> were styled already?

Reply:
Absolutely, it's fixed.